EcoSoapBank/Network Controller/Keychain+Okta.swift

```diff
@@ -22,7 +23,7 @@ extension Keychain {
                date > Date() {
                 return token
             } else {
-                NotificationCenter.default.post(name: .oktaAuthenticationExpired, object: nil)
+                OktaAuth.error.send(LoginError.notLoggedIn)
```

Comment:
Apparently some of the problem was stuff I had done... whoops.

Reply:
This one was more that I wanted to get a more specific error back when something failed, so I added the OktaAuth error passthrough in place of the notifications. I know you can also use notifications for that but then there's all the `Any` and downcasting and blegch.